applications/facemask/configs/config_handler.py

```diff
@@ -33,10 +33,12 @@ def __init__(self, path: str):
         self.CLASSIFIER_NAME = self._config["classifier"]["name"]
         self.CLASSIFIER_MODEL_PATH = self._config["classifier"]["model_path"]
         self.CLASSIFIER_INPUT_SIZE = self._config["classifier"]["input_size"]
+        self.CLASSIFIER_TENSORRT_PRECISION = self._config["classifier"]["tensorrt_precision"]
 
         self.DETECTOR_NAME = self._config["detector"]["name"]
         self.DETECTOR_MODEL_PATH = self._config["detector"]["model_path"]
         self.DETECTOR_INPUT_SIZE = self._config["detector"]["input_size"]
+        self.DETECTOR_TENSORRT_PRECISION = self._config["detector"]["tensorrt_precision"]
```

Comment:
Since other config files do not have this field, the code will break on other devices.